src/test/unit_tests/test_nsfs_glacier_backend.js

```diff
@@ -168,5 +169,30 @@ mocha.describe('nsfs_glacier', async () => {
 			assert(expected_expiry.getMonth() === md.restore_status.expiry_time.getMonth());
 			assert(expected_expiry.getFullYear() === md.restore_status.expiry_time.getFullYear());
 		});
+
+		mocha.it('restore-object should fail restore when days exceed max allowed days', async () => {
+            const data = crypto.randomBytes(100);
+			const params = {
+                bucket: upload_bkt,
+                key: restore_key,
+				storage_class: s3_utils.STORAGE_CLASS_GLACIER,
+                xattr,
+				days: config.NSFS_GLACIER_RESTORE_MAX_DAYS + 1,
+                source_stream: buffer_utils.buffer_to_read_stream(data)
+            };
+
+            const upload_res = await glacier_ns.upload_object(params, dummy_object_sdk);
+            console.log('upload_object response', inspect(upload_res));
+
+			try {
```

Comment:
check your tabs vs spaces?

Reply:
I will shift the blame on VSCode 😆 .

Fixed it.